examples/linear_model/plot_ols_ridge_variance.py

```diff
@@ -44,28 +44,26 @@
 
 fignum = 1
 for name, clf in classifiers.items():
-    fig = plt.figure(fignum, figsize=(4, 3))
+    plt.figure(fignum, figsize=(4, 3))
     plt.clf()
     plt.title(name)
-    ax = plt.axes([.12, .12, .8, .8])
 
     for _ in range(6):
         this_X = .1 * np.random.normal(size=(2, 1)) + X_train
         clf.fit(this_X, y_train)
 
-        ax.plot(X_test, clf.predict(X_test), color='gray')
-        ax.scatter(this_X, y_train, s=3, c='gray', marker='o', zorder=10)
+        plt.plot(X_test, clf.predict(X_test), color='gray')
+        plt.scatter(this_X, y_train, s=3, c='gray', marker='o', zorder=10)
 
     clf.fit(X_train, y_train)
-    ax.plot(X_test, clf.predict(X_test), linewidth=2, color='blue')
-    ax.scatter(X_train, y_train, s=30, c='red', marker='+', zorder=10)
-
-    ax.set_xticks(())
-    ax.set_yticks(())
-    ax.set_ylim((0, 1.6))
-    ax.set_xlabel('X')
-    ax.set_ylabel('y')
-    ax.set_xlim(0, 2)
+    plt.plot(X_test, clf.predict(X_test), linewidth=2, color='blue')
+    plt.scatter(X_train, y_train, s=30, c='red', marker='+', zorder=10)
+
+    plt.ylim((0, 1.6))
+    plt.xlabel('X')
+    plt.ylabel('y')
+    plt.xlim(0, 2)
     fignum += 1
 
+plt.tight_layout()
```

Comment:
Ok figured out the error. Basically by default the x axis label is too low, so it is partially cropped out of the picture generated. However, when you set `tight_layout()` it rescales the plot layout to correctly show the labels.